### PR TITLE
OIDC Deploy for production

### PR DESCRIPTION
**Pull request by Crola1702**
This PR will change the AWS credentials to assume a special role for gazebosim-web-frontent deployment, which only has access to the resources used in this action.

### CODE REVIEW

**Review by azeey — APPROVED** [review]

**Comment by azeey** [comment]
I'll wait for @claraberendsen to approve before merging.

**Review by claraberendsen — APPROVED** [review]
This LGTM, thanks for setting this up @Crola1702!
I can confirm the invalidation was properly done on the console. 
Let's merge and verify that the production action works as expected.

**Comment by Crola1702** [comment]
@azeey, production action needs approval to run :thinking: 

See: https://github.com/gazebo-web/gazebosim-web-frontend/actions/runs/9386669239/job/25847795806

**Pull request merged**
